src/uct/ib/base/ib_md.h

```diff
@@ -333,7 +333,10 @@ uct_ib_md_is_flush_rkey_valid(uint32_t flush_rkey) {
 
 static UCS_F_ALWAYS_INLINE uint8_t uct_ib_md_get_atomic_mr_id(uct_ib_md_t *md)
 {
-    return md->flush_rkey >> 8;
+    if (md->config.enable_indirect_atomic) {
```

Comment:
BTW, why do we need this change? i would expect setting IB_INDIRECT_ATOMIC=n in the test would already disable KSM atomic, w/o additional changes in IB

Reply:
currently `ext.enable_indirect_atomic` is not used at all in the code (some kind of leftover).

Reply:
i see, so maybe we should make this var affect initiator side (send atomic ops to regular MR with offset 0) and not responder side, to avoid breaking remote flush